css/style.css

```diff
@@ -1,20 +1,20 @@
 /* Box Model Hack */
 * {
```

Comment:
Will need to setup a linter / prettier / guidelines on how we're formatting our code so we don't have a bunch of these random conflicts.

Also seems like the previous css file wasn't pulled and there's a bit of missing/old code.